### DISCUSSION TITLE
Request: Using Dune to provide staking dashboard for Darwinia.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#15] [Integration]
This include effort that learn to use, test, integrate with [Dune](https://dune.com/), dune has a large community for data fans and research, and good analytics and visualisation will provide much better understanding and feedback to the community about the on-chain economic and activities. One simple objective would be develop an Darwinia staking dashboard template on Dune for community users further to fork and play.

@furoxr Would you please provide support on this, or directly follow up on this? There could be integration works for dune to support new EVM chains like Darwinia, or learn the best practice for indexing the on-chain data.

Also looking forward to provide a guide for others to adopt dune and develop more data pages.

**Comment by furoxr** [comment]
I'd like to work on this~

**Reply by hackfisher** [reply]
https://feedback.dune.xyz/data-request/p/support-for-darwinia-network-1

**Comment by hackfisher** [comment]
Some other resources from @wuminzhe 

1. https://github.com/blockchain-etl
2. https://dapplooker.com/

**Comment by wuminzhe** [comment]
https://docs.subsquid.io/basics/overview/
It's an indexer with transactional databases (e.g. Postgres) and data lakes (e.g. s3) support. This feature may be used to work with other general data analytics platform like [metabase.com](https://www.metabase.com/).
